web/yo/app/scripts/controllers/gene.js

```diff
@@ -381,6 +402,25 @@ angular.module('oncokbApp')
                     } else return mainUtils.processedInReview('precise', uuid);
                 }
             };
+            $scope.displayPreciselyFire = function(key) {
+                if (!$rootScope.reviewMode) return true;
+                else if (key === 'type') {
+                    // var uuid_ocg = $scope.geneFire.type.ocg_uuid;
```

Comment:
Do nothing here?

Reply:
This will be taken care of in the Review related refactor.